assets/src/dashboard/app/views/editorSettings/components.js

```diff
@@ -50,61 +51,61 @@ export const SettingForm = styled.form`
   column-gap: 6.56%;
   padding-bottom: 52px;
 
-  @media ${({ theme }) => theme.DEPRECATED_THEME.breakpoint.largeDisplayPhone} {
+  @media ${({ theme }) => theme.breakpoint.mobile} {
     grid-template-columns: 100%;
     row-gap: 20px;
   }
 `;
 
-export const SettingHeading = styled.h3`
-  ${TypographyPresets.Small};
-  font-weight: ${({ theme }) => theme.DEPRECATED_THEME.typography.weight.bold};
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.black};
-  padding-bottom: 8px;
-`;
-
-export const FormContainer = styled.div`
-  input {
-    width: 100%;
-    height: 32px;
-  }
+export const SettingHeading = styled(Headline).attrs({
+  as: 'h3',
+})`
+  ${({ theme }) =>
+    themeHelpers.expandPresetStyles({
+      preset: {
+        ...theme.typography.presets.label[
+          THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.LARGE
+        ],
+      },
+      theme,
+    })};
+  margin: 8px 0;
 `;
 
 export const InlineLink = styled(Link)`
-  margin-left: 0.25em;
+  display: inline-block;
 `;
 
-export const HelperText = styled.p`
-  ${TypographyPresets.Small};
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray200};
+export const HelperText = styled(Text)`
+  color: ${({ theme }) => theme.colors.fg.tertiary};
 `;
 
-export const DefaultLogoText = styled.p`
-  ${TypographyPresets.ExtraSmall};
-  margin: 0;
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray200};
-  width: 100%;
+export const CenterMutedText = styled(Text)`
+  color: ${({ theme }) => theme.colors.fg.tertiary};
   text-align: center;
 `;
+
+export const SettingSubheading = styled(HelperText)`
+  padding: 8px 0;
+`;
+
 export const TextInputHelperText = styled(HelperText)`
-  padding-top: 10px;
+  padding-top: 12px;
 `;
 
-export const FinePrintHelperText = styled.p`
-  ${TypographyPresets.ExtraSmall};
-  padding-top: 10px;
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray200};
+export const CheckboxLabel = styled(Text)`
+  display: flex;
+  justify-content: flex-start;
+  margin-top: 8px;
 `;
 
-export const FormLabel = styled.span`
-  ${TypographyPresets.ExtraSmall};
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray400};
+export const CheckboxLabelText = styled(HelperText)`
+  margin-left: 8px;
 `;
 
-export const Error = styled.p`
-  ${TypographyPresets.ExtraSmall};
+export const Error = styled(CenterMutedText)`
```

Comment:
This error is used in file upload, but the other uses of error were replaced with the input's hint.